examples/MagicWeather/app/src/main/java/com/revenuecat/sample/data/Constants.kt

```diff
@@ -7,15 +7,15 @@ object Constants {
 
      Modify this property to reflect your Google Play app's API key.
      */
-    const val GOOGLE_API_KEY = "googl_api_key"
+    const val GOOGLE_API_KEY = "goog_cRtaFpfwfcQYPXkIEqFUkVszUfN"
```

Comment:
Oops

Reply:
Pushed it on purpose for testers to be able to run it